hermes/utils/book_crossing_etl/bookcrossing.py

```diff
@@ -257,8 +257,8 @@ def parse_book_line(line):
 
     with\
         open(args.ratings, 'rb') as csvfile,\
-        open("book-crossing_implicit_ratings.json", 'w') as imp,\
-        open("book-crossing_explicit_ratings.json", 'w') as exp:
+        open("implicit_ratings.json", 'w') as imp,\
```

Comment:
Yes, I noticed it too that the git config rerere did not pick up the change. It probably occurred because of the conflict in directory name. The next commit should reflect this fix. Thanks for catching it.